--- /projects/caeg/scratch/kbd606/tmp/af_orig/extracted/alphafold/model/config.py	2026-03-16 14:16:18.000000000 +0100
+++ /projects/caeg/people/kbd606/opt/colabfold-env/lib/python3.10/site-packages/alphafold/model/config.py	2026-05-10 06:51:55.977742000 +0200
@@ -380,7 +380,7 @@
             'bfloat16_output': False,
             'deterministic': False,
             'multimer_mode': False,
-            'subbatch_size': 4,
+            'subbatch_size': 256,
             'use_remat': False,
             'zero_init': True,
             'eval_dropout': False,
@@ -618,7 +618,7 @@
             'bfloat16_output': False,
             'deterministic': False,
             'multimer_mode': True,
-            'subbatch_size': 4,
+            'subbatch_size': 256,
             'use_remat': False,
             'zero_init': True,
             'eval_dropout': False,
